modules/layers/src/base-tile-layer/base-tile-layer.js

```diff
@@ -70,7 +70,10 @@ export default class BaseTileLayer extends CompositeLayer {
     }
 
     const {viewport} = context;
-    if (changeFlags.viewportChanged && viewport.id !== 'DEFAULT-INITIAL-VIEWPORT') {
+    if (
+      createNewCache ||
```

Comment:
I don't have enough context to know the impact of this change, but it might make sense to add something about it to the PR description for future reference.

Reply:
This is a part of another pending PR #4139. I plan to get that PR landed first so that we can drop this.